specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2019-01-01-preview/cpimTenant.json

```diff
@@ -0,0 +1,917 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "B2CTenants",
+    "version": "2019-01-01-preview",
+    "description": "Manage the Azure resource for an Azure Active Directory B2C tenant."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AzureActiveDirectory/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "B2CTenants"
+        ],
+        "operationId": "B2CTenants_CheckNameAvailability",
+        "description": "Checks the availability and validity of a domain name for the tenant.",
+        "x-ms-examples": {
+          "Check name availability - available": {
+            "$ref": "./examples/checkNameAvailability-available.json"
+          },
+          "Check name availability - taken": {
+            "$ref": "./examples/checkNameAvailability-taken.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "checkNameAvailabilityRequestBody",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The result of checking for the name availability.",
+            "schema": {
+              "$ref": "#/definitions/NameAvailabilityResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureActiveDirectory/b2cDirectories": {
```

Comment:
>b2cDirectories [](start = 113, length = 14)

This looks like completely new set of capabilities; please confirm if you've a formal review meeting with ARM API Review board and if yes please share the notes with armapireview@microsoft.com to take the review forward.

Reply:
Already reviewed with ARM (see other PR)